src/styles/pages/product-listing.scss

```diff
@@ -0,0 +1,170 @@
+@import "../scaffold.scss";
+
+.product-listing {
+  padding-top: 0;
+
+  @include media(">=notebook") {
+    padding-top: var(--page-padding-notebook);
+  }
+}
+
+.product-listing__breadcrumb {
+  padding-top: var(--space-3);
+  padding-bottom: var(--space-3);
+}
+
+// Temporary till we have skeletons.
+.product-listing__data-loading {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding-top: var(--space-5);
+}
+
+.product-listing__hero {
+  width: 100%;
+}
+
+.product-listing__content-grid {
+  display: flex;
+  flex-direction: column;
+  padding-right: 0;
+  padding-left: 0;
+
+  @include media(">=notebook") {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    padding-right: var(--page-padding-notebook);
+    padding-left: var(--page-padding-notebook);
+    column-gap: var(--space-4);
+    row-gap: var(--space-0);
+  }
+}
+
+.product-listing__filters {
+  display: none;
+
+  @include media(">=notebook") {
+    position: sticky;
+    top: var(--grid-gap-0);
+    display: flex;
+    grid-column: 1 / span 3;
+    align-items: center;
+  }
+}
+
+.product-listing__filters-bar {
+  @include media(">=notebook") {
+    position: sticky;
+    top: rem(60px);
+    grid-column: 1 / span 3;
+    align-self: start;
+  }
+}
+
+.product-listing__sort {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  order: 1;
+  width: 100%;
+  min-height: 68px;
```

Comment:
```suggestion
  min-height: rem(68px);
```

Reply:
Applied it [here](https://github.com/vtex-sites/base.store/pull/281/commits/f0e1e8e5bea8672a898a0a0da68ceab4c69bbd23)! Thanks!